Bottom: 2c622f2468d9e3b0d73e43039eb1048917f38b2d
Top:    51864b089669e4e87a1c4ca0cf82050f6bad16d0
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-20 23:38:16 +0100

Refresh of major-re-write-to-drawing-rout

---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index 990e744..325b7b4 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -1619,8 +1619,6 @@ ghid_port_drawing_area_expose_event_cb (GtkWidget * widget,
   int min_depth;
   int max_depth;
 
-//  if (!one_shot) return;
-
   /* make GL-context "current" */
   if (!gdk_gl_drawable_gl_begin (pGlDrawable, pGlContext)) {
     return FALSE;
